documentation/book/proc-manual-restart-zookeeper.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// assembly-todo.adoc
+
+[id='proc-manual-restart-zookeeper-{context}']
+= Manually restart a Zookeeper cluster
+
+This procedure describes how to manually restart an existing Zookeeper cluster by using an {ProductPlatformName} annotation.
+
+.Prerequisites
+
+* A running Zookeeper cluster.
+* A running Cluster Operator.
+
+.Procedure
+
+. Find the name of `StatefulSet` which controls Zookeeper pods to restart.
++
+Lets say the cluster is named _my-cluster_. The name of the Zookeeper StatefulSet is _my-cluster-zookeeper_.
++
+
+. Annotate `StatefulSet` resource in {ProductPlatformName}.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl annotate statefulset _cluster-name_-zookeeper cluster.operator.strimzi.io/manual-rolling-update=true
+endif::Kubernetes[]
++
+On {OpenShiftName} this can be done using `oc annotate`:
+[source,shell,subs=+quotes]
+oc annotate statefulset _cluster-name_-zookeeper cluster.operator.strimzi.io/manual-rolling-update=true
++
+. Wait for next reconciliation and for all pods in annotated `StatefulSet` rolling update will be done.
+After all pods are restarted the annotation is removed from `StatefulSet`.
+
+
+.Additional resources
+
+* For more information about deploying the Cluster Operator, see xref:cluster-operator-str[].
+* For more information about deploying the Zookeeper cluster, see xref:deploying-zookeeper-cluster-openshift-str[] and xref:deploying-zookeeper-cluster-kubernetes-str[].
```

Comment:
There is no errors on the build now because these adoc files aren't included (and you should talk with @laidan6000 for making a decision) but this reference doesn't exist `deploying-zookeeper-cluster-openshift-str`. You have to change to `deploying-kafka-cluster-openshift-str` because we don't have a real Zookeeper deployment doc (it's all in the Kafka stuff).

Reply:
Thanks @ppatierno